src/utils.js

```diff
@@ -1,20 +1,12 @@
 'use strict';
 
+var isError = require('iserror');
+
 function isObject(what) {
     return typeof what === 'object' && what !== null;
 }
 
-// Sorta yanked from https://github.com/joyent/node/blob/aa3b4b4/lib/util.js#L560
-// with some tiny modifications
-function isError(what) {
-    var toString = {}.toString.call(what);
-    return isObject(what) &&
```

Comment:
As commented: https://github.com/joyent/node/blob/aa3b4b4/lib/util.js#L560

This was just yanked from Node core, so I'm not entirely sure what this is doing. Just standing on top of giants here.